future/src/ct/ct_export2html.cc

```diff
@@ -666,9 +678,11 @@ bool has_pandoc()
 }
 
 
-void to_html(std::istream& input, std::ostream& output) 
+void to_html(std::istream& input, std::ostream& output, std::string from_format /* = "markdown" */)
```

Comment:
it's not necessary
but it may be unclear that `to_html` implicitly uses `markdown` style. maybe it would be better if format will be explicit

Reply:
Pandoc defaults to markdown thats why it's needed, I didn't make it explicit since the expected behaviour of pandoc if you do not specify a format is markdown but I can see how that could be unclear xD